doc/help/kdb-cache.md

```diff
@@ -19,11 +19,9 @@ subcommand can only remove a user's cache files (i.e. not system wide).
 ## OPTIONS
 
 - `-H`, `--help`:
-  Show the man page.
+  Show usage of command.
 - `-V`, `--version`:
   Print version info.
-- `-p`, `--profile <profile>`:
```

Comment:
Are profiles not supported anymore? This isn't mentioned in the readme. Please write an issue with all features that are missing.

Reply:
I'll mention it in the readme and add an issue(#4965). I think this was discussed a while ago, but maybe I misremember, regardless, it should be mentioned.